src/rpc/blockchain.cpp

```diff
@@ -2247,6 +2247,106 @@ static UniValue getblockfilter(const JSONRPCRequest& request)
     return ret;
 }
 
+/**
+ * Serialize the UTXO set to a file for loading elsewhere.
+ *
+ * @see SnapshotMetadata
+ */
+UniValue dumptxoutset(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 1)
+        throw std::runtime_error(
+            RPCHelpMan{
+                "dumptxoutset",
+                "\nWrite the serialized UTXO set to disk.\n"
+                "Incidentally flushes the latest coinsdb (leveldb) to disk.\n",
+                {
+                    {"path",
+                        RPCArg::Type::STR,
+                        RPCArg::Optional::NO,
+                        /* default_val */ "",
+                        "path to the output file. If relative, will be prefixed by datadir."},
+                },
+                RPCResults{},
+                RPCExamples{"{\n"
+            "  \"coins_written\": n,   (numeric) the number of coins written in the snapshot\n"
+            "  \"base_hash\": \"...\",   (string) the hash of the base of the snapshot\n"
+            "  \"base_height\": n,     (string) the hash of the base of the snapshot\n"
+            "  \"path\": \"...\"         (string) the absolute path that the snapshot was written to\n"
```

Comment:
Looks like this is the Result, not an Example?

Reply:
Ah, right - fixed.